v2/emailpassword/common-customizations/email-verification/about.mdx

```diff
@@ -128,9 +128,11 @@ init(
 <TabItem value="reactjs">
 
 ```tsx
-import SuperTokens from "supertokens-auth-react";
+import SuperTokens, { SuperTokensWrapper } from "supertokens-auth-react";
 import EmailVerification from "supertokens-auth-react/recipe/emailverification";
+import { EmailVerificationPreBuiltUI } from "supertokens-auth-react/recipe/emailverification/preBuiltUI";
```

Comment:
Im not sure if its a good idea to have import paths have camel casing. There are OS in which it ignores this and then it may not find the file. We should change the file to `ui` instead

Reply:
The way an OS ignores this is by using case-insensitive path matching. This means that we'll only have issues if we add a lowercase preBuiltUI later next to it or change the casing.